modules/ROOT/pages/introduction-to-tiny-comments.adoc

```diff
@@ -37,6 +37,20 @@ The Comments plugin allows the user to perform the following functions:
 * Resolve a comment thread
 * Delete a comment or comment thread
 
+=== Supported content
+
+The Comments plugin allows the user to comment on text.
+
+In addition, the user can comment on various blocks, including:
+
+NOTE: This feature is only available for {productname} 6.1 and later.
```

Comment:
This shouldn't be done like this... we should create a partial and include it instead. See https://github.com/tinymce/tinymce-docs/blob/release/TinyMCE_5.10/modules/ROOT/partials/misc/requires_5_9v.adoc for an example. Additionally, this is quite odd being between `In addition, the user can comment on various blocks, including:` which is the start of the list and the actual list.

- [ ] Fix admonition warning
- [ ] Move above start of list instead of in the middle

Reply:
Admonition partial has been added. I also reworked the structure of the supported content section so it now looks like this:
![image](https://user-images.githubusercontent.com/46954949/178429716-f402d6b9-1d2e-461f-b87c-c56e7f578710.png)

Reply:
Hmm... I think we need that admonition above the list if we're using the above.

Reply:
Should be better now. Thanks for that